foliant/config/path.py

```diff
@@ -12,9 +13,23 @@ def _resolve_path_tag(self, _, node) -> str:
         '''
 
         path = Path(node.value).expanduser()
-        return (self.project_path/path).resolve(strict=True).as_posix()
+        return (self.project_path / path).resolve(strict=True).as_posix()
+
+    def _resolve_project_path_tag(self, _, node) -> str:
+        '''Convert value after ``!project_path`` to Path object relative to the
+        project path.
+        '''
+
+        return (self.project_path / node.value).absolute()
```

Comment:
Why not `resolve()` but not strict?

https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve

Reply:
Agreed. According to docs, `resolve` also offers normalization, which `absolute` does not

Reply:
> Convert value after ``!project_path`` to Path object relative to the project path.

> return (self.project_path / node.value).absolute()

Why is relative path returned by `absolute` function?

Reply:
@moigagoo fixed the docs!
This method takes path after `!project_path` tag and makes it absoulte (considering this path being relative to project root)